addons/website_sale/static/src/js/notification/cart_notification/cart_notification.js

```diff
@@ -9,7 +9,7 @@ export class CartNotification extends Component {
     static template = "website_sale.cartNotification";
     static props = {
         message: [String, { toString: Function }],
-        warning: [String, { toString: Function }],
+        warning: {type : [String, { toString: Function }],optional: true},
         lines: {
             type: Array,
             element: {
```

Comment:
```suggestion
            type: Array,
            optional: true,
            element: {
```

Reply:
Hello, Thanks for the review. I added the optional : true  on the lines :)